src/revlanguage/workspace/FunctionTable.cpp

```diff
@@ -305,7 +305,7 @@ const Function& FunctionTable::findFunction(const std::string& name, const std::
         }
         else
         {
-            throw RbException("No function named '"+ name + "'");
+            return nullptr;
```

Comment:
Why does this now return a nullptr instead of the error message?

Reply:
I added the following comment:
```
            // Here we just return a nullptr to indicate failure.
            // The caller can produce better error messages, so it is the caller's job to throw an exception.
```